### DISCUSSION TITLE
DASimpleFoamAD.py not passing the tests

### DISCUSSION THREAD

**Discussion by justAPhDStudent** [mdolab/dafoam#77] [Tutorials]
Hello,
I could install the dafoam platform correctly on Ubuntu 18.04; as mentioned in the documentation. The test for DASimpleFoamAD.py is not passing with following error:
    from .pyDASolverIncompressibleAD import pyDASolvers as pyDASolversAD
ModuleNotFoundError: No module named 'dafoam.pyDASolverIncompressibleAD'

What can be the possible cause ? All other tests for compressible, solids are passing.
Best regards,
Tom

**Comment by friedenhe** [comment]
We have recently updated the code to v2.2.0 with a new capability to compute matrix-free Jacobian adjoint. However, we haven't updated the installation guide yet (need to compile an OpenFoam version differentiated by CodiPack). For now, just comment out `runTests DASimpleFoamAD` in `tests/Allrun`, and you should be able to run the reg tests. We will add the installation guide soon.
